### PR TITLE
Test dask wih npartitions

### PR DESCRIPTION
**Pull request by Nikoleta-v3**
## What type of PR is this? (check all applicable)

- [ ] 💾 Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 🔧 Optimization
- [ ] 📝 Documentation
- [x] ✅ Test
- [ ] 🐳 Other

## Related issues 

- Related issue # 
- Closes #

## Checklist

- [x] Code follows style guide (ruff)
- [ ] Tests added 
- [ ] Documented the changes

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def polars_lazy_constructor(obj: Any) -> pl.LazyFrame:
 
 def dask_lazy_constructor(obj: Any) -> IntoFrame:  # pragma: no cover
     dd = get_dask_dataframe()
-    return dd.from_pandas(pd.DataFrame(obj))  # type: ignore[no-any-return]
+    return dd.from_pandas(pd.DataFrame(obj), npartitions=2)  # type: ignore[no-any-return]
```
Comment (MarcoGorelli) [inline tests/conftest.py:77]: let's temporarilly revert this until we've fixed up all tests to work with npartitions=2
```suggestion
    return dd.from_pandas(pd.DataFrame(obj))  # type: ignore[no-any-return]
```

**Comment by Nikoleta-v3** [comment]
Hey @MarcoGorelli 👋🏻 sorry for the failure. I started working on the `tests/expr_and_series/over_test.py` file but never finished the changes and thus the failure. The tests now pass locally with `pytest` 😄

**Review by MarcoGorelli — APPROVED** [review]
amazing, thanks so much @Nikoleta-v3 , well done!

**Comment by Nikoleta-v3** [comment]
Thank you @MarcoGorelli! I'm happy to keep working on the issue if it's useful. It's been a great introduction to the package, haha 😄

**Comment by MarcoGorelli** [comment]
definitely useful, thanks! any help towards being able to have `npartitions=2` in `conftest.py` (or towards anything that interests you) is definitely welcome🙏 🤗 

we've got a friendly discord too https://discord.gg/V3PqtB4VA4

**Pull request merged**